packages/plugin-core/src/_extension.ts

```diff
@@ -166,6 +167,14 @@ export async function _activate(
       sessionId: AnalyticsUtils.getSessionId(),
       release: AnalyticsUtils.getVSCodeSentryRelease(),
     });
+
+    // Temp: store the user's anonymous ID into global state so that we can link
+    // local ext users to web ext users:
+    context.globalState.setKeysForSync([GLOBAL_STATE_KEYS.ANONYMOUS_ID]);
```

Comment:
we have a state service to set state > [[../packages/plugin-core/src/services/stateService.ts]]
can we undeprecate it and put it in there instead?

Reply:
we have a state service to set state > [[../packages/plugin-core/src/services/stateService.ts]]
can we undeprecate & implement the logic from there?

Reply:
I'd rather not use State Service because it's a singleton. I think having a enum of key values is enough.